tests/testthat/test-reprex.R

```diff
@@ -11,7 +11,7 @@ test_that("reprex() works with code that deals with srcrefs", {
     input = "utils::getParseData(parse(text = 'a'))\n",
     advertise = FALSE
   )
-  expect_known_output(print(ret), test_path("reference/srcref_reprex"))
+  expect_snapshot_output(print(ret))
```

Comment:
I struggled with this one. `expect_snapshot_value()` seems like the correct function, but the fact that the output (a character vector) holds markdown causes it to barf?

```
Error: parse error: trailing garbage
                             [1] "``` r"                              
                     (right here) ------^

Backtrace:
 1. testthat::expect_snapshot_value(...) test-reprex.R:10:2
 2. testthat:::expect_snapshot_helper(...)
 3. snapshotter$take_snapshot(val, save = save, load = load, ...)
 4. testthat:::load(old_raw)
 5. jsonlite::fromJSON(x, simplifyVector = FALSE)
 6. jsonlite:::parse_and_simplify(...)
 7. jsonlite:::parseJSON(txt, bigint_as_char)
 8. jsonlite:::parse_string(txt, bigint_as_char)
 9. ```

Reply:
There is no choice for `style = c("json", "json2", "deparse", "serialize")` that makes `expect_snapshot_value()` work here.

Reply:
Can you give me a simple reprex? It's likely that I've forgotten to escape something.

Reply:
https://github.com/r-lib/testthat/issues/1294